imagetest/test_suites/network/setup.go

```diff
@@ -73,7 +73,17 @@ func TestSetup(t *imagetest.TestWorkflow) error {
 		return err
 	}
 
+	vm3, err := t.CreateTestVM("vm3")
+	if err != nil {
+		return err
+	}
+	vm3.ChangeNicTypeToGVNIC()
+	if err := vm3.Reboot(); err != nil {
+		return err
+	}
+
 	vm1.RunTests("TestPingVMToVM|TestDHCP|TestDefaultMTU")
 	vm2.RunTests("TestAlias")
+	vm3.RunTests("TestGVNIC")
```

Comment:
just to clarify - you only want to run this one test on the GVNIC-enabled instance now, right? Would it be possible to bundle it with either vm1 or vm2 to reduce resource usage?

Reply:
I can actually however since other tests test specific network configuration will using GVNIC affect those tests output in any way?